shared/src/packages/sites/libs/validation-schemas/site-get-by-project.validation-schema.ts

```diff
@@ -0,0 +1,9 @@
+import joi from 'joi';
+
+import { type SiteGetByProjectRequestDtoType } from '../types/types.js';
+
+const siteGetByProject = joi.object<SiteGetByProjectRequestDtoType, true>({
```

Comment:
```suggestion
const siteGetByProjectParams = joi.object<SiteGetByProjectRequestDtoType, true>({
```
don't forget about file name